tests/NuGet.Services.AzureSearch.FunctionalTests/Relevancy/V3RelevancyFunctionalTests.cs

```diff
@@ -14,65 +16,74 @@ public V3RelevancyFunctionalTests(CommonFixture fixture, ITestOutputHelper testO
         {
         }
 
-        [RelevancyFact]
-        public async Task Json()
+        [RelevancyTheory]
+        [MemberData(nameof(EnsureFirstResultsData))]
+        public async Task EnsureFirstResults(string searchTerm, string[] expectedFirstResults)
         {
-            var results = await SearchAsync("json");
+            var results = await SearchAsync(searchTerm);
 
-            Assert.True(results.Count > 2);
-            Assert.Equal("json", results[0]);
-            Assert.Equal("newtonsoft.json", results[1]);
-        }
-
-        [RelevancyFact]
-        public async Task NewtonsoftJson()
-        {
-            var results = await SearchAsync("Newtonsoft.Json");
+            Assert.True(results.Count > expectedFirstResults.Length);
 
-            Assert.NotEmpty(results);
-            Assert.Equal("newtonsoft.json", results[0]);
+            for (var i = 0; i < expectedFirstResults.Length; i++)
+            {
+                Assert.True(expectedFirstResults[i] == results[i], $"Expected result '{expectedFirstResults[i]}' at index #{i} for query '{searchTerm}'");
+            }
         }
 
-        [RelevancyFact]
-        public async Task Log()
+        public static IEnumerable<object[]> EnsureFirstResultsData()
         {
-            var results = await SearchAsync("Log");
+            // Test that common queries have the most frequently selected results at the top.
+            // These results were determined using the "BrowserSearchPage" and "BrowserSearchSelection" metrics
+            // on the Gallery's Application Insights telemetry.
+            // TODO: Reduce exact match boost and update "json.net", "entity framework", "redis", and "csv" tests.
+            // See: https://github.com/NuGet/NuGetGallery/issues/7330
+            yield return new object[] { "newtonsoft.json", new[] { "newtonsoft.json" } };
+            yield return new object[] { "newtonsoft", new[] { "newtonsoft.json" } };
+            yield return new object[] { "json.net", new[] { "json.net", "newtonsoft.json" } };
+            yield return new object[] { "json", new[] { "json", "newtonsoft.json" } };
 
-            Assert.NotEmpty(results);
-            Assert.Contains("log4net", results);
+            yield return new object[] { "tags:\"aws-sdk-v3\"", new[] { "awssdk.core", "awssdk.s3" } };
```

Comment:
did you run this test against dev? I do not see a package "awssdk.s3" in search(at least in gallery)

Reply:
oh are these tests only meant for prod? forgot.